test/unit/calculators/marriage_abroad_calculator_test.rb

```diff
@@ -938,6 +938,22 @@ class MarriageAbroadCalculatorTest < ActiveSupport::TestCase
           refute @calculator.ceremony_country_offers_pacs?
         end
       end
+
+      context '#french_overseas_territory_offering_pacs?' do
+        setup do
+          @calculator = MarriageAbroadCalculator.new
+        end
+
+        should 'return true if a PACS is available in the ceremony country and the country is a French overseas territory' do
+          @calculator.ceremony_country = 'new-caledonia'
+          assert @calculator.french_overseas_territory_offering_pacs?
+        end
+
+        should "return false if PACS is available in the ceremony country but it's not a French overseas territory" do
+          @calculator.ceremony_country = 'france'
+          refute @calculator.french_overseas_territory_offering_pacs?
+        end
+      end
```

Comment:
I've amended this commit in https://github.com/alphagov/smart-answers/pull/2470/commits/1a316a1b8d8f6b4c2b8733750c588559937263bc - hopefully that makes the naming of these tests clearer.